src/uct/ib/base/ib_md.c

```diff
@@ -1267,7 +1267,7 @@ void uct_ib_md_check_odp(uct_ib_md_t *md)
     }
 
     md->reg_nonblock_mem_types = UCS_BIT(UCS_MEMORY_TYPE_HOST);
-    ucs_debug("%s: ODP is supported, version 1", device_name);
+    ucs_debug("%s: ODP is supported", device_name);
```

Comment:
why removed `version 1`?

Reply:
because in Verbs MD we don't really know if the FW supports ODPv1 or ODPv2

Reply:
then seems we do not need to mention it in new `uct_ib_mlx5dv_md_check_odp` fucntion

Reply:
fixed by force-push